### PR TITLE
Added a note

### PR DESCRIPTION
**Pull request by joinimran**
As per the author's recommendation, I have updated a note in the document.

Problem: https://github.com/MicrosoftDocs/microsoft-365-docs/issues/486

### CODE REVIEW

**Comment by mypil** [comment]
@nenonix @JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: microsoft-365/compliance/prevent-email-from-being-marked-as-spam.md [hunk 1]
```diff
@@ -22,6 +22,9 @@ description: "Learn how to prevent False Positives and keep real email out of ju
 
 If you get a false positive, you should report the message to Microsoft by using the [Use the Report Message add-in](https://support.office.com/article/b5caa9f1-cdf3-4443-af8c-ff724ea719d2). Additionally, you can submit the message using [Submissions Explorer](/security/office-365-security/admin-submission.md).
 
+> [!NOTE]
+> Anyother fields in header that are not mentioned below are exclusively used by the Microsoft anti-spam team for diagnostic purposes.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/prevent-email-from-being-marked-as-spam.md]:
```suggestion
> Any other fields in the header that are not mentioned below are exclusively used by the Microsoft anti-spam team for diagnostic purposes.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy Review

**Comment by joinimran** [comment]
Ready to go live.

**Comment by mypil** [comment]
@chrisda - PR has been copy edited and is ready for final review. This is based on this issue https://github.com/MicrosoftDocs/microsoft-365-docs/issues/486#issuecomment-540828898

Thank you.

cc: @kenwith

**Pull request merged**